cryptol-remote-api/python/cryptol/cryptoltypes.py

```diff
@@ -450,6 +457,8 @@ def to_type(t : Any) -> CryptolType:
         return Tuple(*map(to_type, t['contents']))
     elif t['type'] == 'record':
         return Record({k : to_type(t['fields'][k]) for k in t['fields']})
+    # elif t['type'] == 'newtype':
+    #     return Newtype(t['name'], t['uid'], map(to_type, t['arguments']))
```

Comment:
IIUC, now it's only possible to write newtypes as concrete syntax, and that's why this alternative is no longer relevant. Is that right? And, if so, should we just remove it rather than commenting it out?

Reply:
Yes, I think that's basically it --  it's either concrete syntax from the client which get's it's meaning/type/etc there or it's an opaque value sent from the server which already has a known type (and whose known type may not be representable syntactically for the same reasons we've introduced the framework for sending back opaque newtype values to clients). I've deleted the commented out code as you suggested 👍